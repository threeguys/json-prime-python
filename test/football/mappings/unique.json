{
    "sorted": { "unique": "$[*].country_name" }
}
