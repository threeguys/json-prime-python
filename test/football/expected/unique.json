[ "England", "International", "Spain", "United States of America" ]
